README.md

```diff
@@ -24,29 +24,39 @@ However, if you'd like you can also build SwiftFrame from source. To do this clo
 -   change into the directory: `cd SwiftFrame`
 -   run the install script: `sh install.sh`
 
-## Usage
+## Rendering Images
 
 To use SwiftFrame, you need to pass it a configuration file (which is a JSON or YAML file):
 
 ```bash
 swiftframe path/to/your/config/file --verbose
 ```
 
-### Available Options
+### Opt-In Flags
+
+You can use the `--help` flag to print SwiftFrame's documentation out. The following flags are available
 
 #### --verbose
 
 Prints additional information during processing and rendering. This will also ask you to confirm the processed config file before SwiftFrame starts rendering
 
-#### --no-color-output
+#### --manual-validation
+
+Pauses after parsing the config file to let you verify the contents
+
+#### --output-whole-image
+
+Outputs the whole image canvas into the output directories before slicing it up into the correct screenshot sizes. Helpful for troubleshooting
+
+### Opt-Out Flags
 
-Disables colored output. Useful for CI logs for example
+#### --color-output/--no-color-output
 
-#### --no-manual-validation
+Controls whether logged output will be colored or now. Useful for CI logs for example. Default is **true**
```

Comment:
µ-nit: "Colored or NOT"

Reply:
I kind of wonder if it would make more sense to just have one flag which is the non-default? Not sure if I'm understanding correctly, but the `--color-output` would not do anything?

Reply:
Hm yes that's true. I think my main reason for having these opt-out flags was the naming of the properties in code it would allow. But I'm fine with inverting the name and simply use `!` before passing them to the `ConfigProcessor`

Reply:
I see yeah. Doesn't make too much of a difference I suppose. I don't really have a strong opinion on this so I'll refer to your judgement :)